docs/modules/plugins/pages/plugin-json.adoc

```diff
@@ -10,24 +10,52 @@ The plugin provides a set of actions for transformation and validation of JSON d
 implementation(group: 'org.vividus', name: 'vividus-plugin-json', version: '{current-version}')
 ----
 
+== xref:commons:variables.adoc[Dynamic variables]
+
+=== JSON context
+The variable provides the HTTP response body of the latest executed HTTP call
+by default or the current JSON element for steps iterating over JSON.
+
+[source,gherkin]
+----
+${json-context}
+----
+
+.Validate the HTTP response matches the regular expression
+[source,gherkin]
+----
+When I send HTTP GET to the relative URL '/get?productsPerPage=5'
+Then number of JSON elements from `${json-context}` by JSON path `$.product` is equal to 5
+----
+
 == Steps
 
 :json-path: https://github.com/json-path/JsonPath#path-examples[JSON Path]
 
 The steps syntax uses two internal (VIVIDUS-only) terms:
 
- * "JSON element" - any part of JSON document including both compex data
-   structures like `array`, `object` and primitive values like `string` in
+ * "JSON element" - any part of JSON document including both complex data
+   structures like `array`, `object` and scalar values like `string` in
    double quotes, `number`, `boolean` (`true` or `false`) and `null`.
- * "JSON element value" or "value of JSON element" - primitive values like
+ * "JSON element value" or "value of JSON element" - scalar values like
    `string` _not wrapped_ into double quotes, `number`, `boolean`
    (`true` or `false`) and `null`.
 
-NOTE: JSON context is pointed to the latest HTTP response by default.
-
 === Save JSON element value from context
 
-Saves a value of JSON element found in JSON context into the variable with
+[WARNING]
+====
+The step is deprecated and will be removed in VIVIDUS 0.6.0. The replacement is
```

Comment:
why do we need deprecated step docs?

Reply:
yes,
- what if someone uses this step, how to understand what it does
- what if there is a link to this doc, we should keep it for a while

Reply:
imho
- if the steps is already used in scenario then it's faster to check it in javadocs
- if end user is not aware of this step and has intent to save a value from JSON he will probably open the docs and find new step by keywords